airbyte-integrations/connectors/destination-chroma/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM python:3.9.11-alpine3.15 as base
```

Comment:
Could you update the Dockerfile to match the one from Milvus? https://github.com/airbytehq/airbyte/pull/29790/files#diff-5973bc6a06b7761850b06571c2506c428b7842816f928ff31801e48af5cc07f1